### DISCUSSION TITLE
How fast?

### DISCUSSION THREAD

**Discussion by pakdev** [Nicholasli1995/EgoNet#11] [Q&A] [answered]
How quickly does inference run on both CPU/GPU?

**Comment by Nicholasli1995** [comment, marked as answer]
CPU is not supported and tested.

Per the current parameters,  a forward pass takes 60-90ms on a 12GB Titan Xp and 23 instances can be processed in parallel. On average, KITTI val set contains 4.2 cars/image (<23), resulting in 60-90ms/image.

To further improve the efficiency, consider using smaller network/model compression/quantization.
